cmd/guacone/cmd/queryVulnerability.go

```diff
@@ -52,7 +52,7 @@ type queryOptions struct {
 }
 
 var queryVulnCmd = &cobra.Command{
-	Use:   "query_vuln [flags]",
+	Use:   "vuln [flags]",
```

Comment:
Is purl required here? If so, It would be good to move that to be a positional arg to be consistent with the others.

Reply:
so the purl is required but the vulnerability ID is not. So keep `vulnerabilityID` as a flag and move purl to a positional arg?

Reply:
(can be in a followup)

Reply:
based on discussion offline, moving purl to a positional arg and keeping `vulnerabilityID` as a flag

Reply:
done.